src/legacy/ui/public/vis/editors/default/agg_params.html

```diff
@@ -33,4 +33,15 @@
   </p>
 </div>
 
+<vis-editor-agg-select
+  agg="agg"
+  label="label"
+  on-change-agg-type="onChangeAggType"
+  agg-type-options="aggTypeOptions"
+  is-select-valid="isSelectValid"
+  ng-model="agg.type"
+  name="aggType"
+  required
```

Comment:
To `ngForm` reacts to changing of the selected value, I added `ng-model` and `required` here. (`name` is used to trigger validation from the link function)